src/pages/create.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import CreateEvent from '../components/create-event/CreateEvent';
+import { getUser } from '../auth';
+import { LocationContext, UserContext } from '../contexts';
+import Header from '../components/Header';
 
-const center = {
-  lng: -90.06911208674771,
-  lat: 29.954767355989652,
+export const getServerSideProps = async function getServerSideProps(context) {
+  const user = await getUser(context);
+  return { props: { user } };
```

Comment:
@ygroener @c4mino the server is assigning your props for you using `getServerSideProps`, it runs all the code in the export block whenever the page loads.